dts/arm/nordic/nrf51822.dtsi

```diff
@@ -153,3 +154,12 @@
 &nvic {
 	arm,num-irq-priority-bits = <2>;
 };
+
+&sw_pwm {
+	timer-instance = <1>;
+	channel-count = <4>;
```

Comment:
`<3>`
4 is the total number of compare channels in TIMER1.
Same applies to TIMER2 used for nRF52s.

Reply:
Of course, changed.